miden/examples/fib_with_merkle_store/fib.masm

```diff
@@ -0,0 +1,6 @@
+begin
+    # This code computes 1001st Fibonacci number
+    repeat.1000
+        swap dup.1 add
+    end
+end
```

Comment:
Would be good if the example illustrates how we can read from various Merkle tree structures which which we instantiated the advice provider (e.g., using `mtree_read` instruction).